src/components/v5/common/MemberSignatureList/MemberSignatureList.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { FC } from 'react';
+
+import { notNull, notUndefined } from '~utils/arrays';
+import { SpinnerLoader } from '~shared/Preloaders';
+import MemberSignature from '../MemberSignature';
+import { MemberSignatureListProps } from './types';
+
+const displayName = 'v5.common.MemberSignatureList';
+
+const MemberSignatureList: FC<MemberSignatureListProps> = ({
+  items,
+  isLoading,
+  title,
+  checkedUsersList,
+}) => {
+  // @TODO: Use this data in recovery mode signatures list component
+  // const { loading, usersWithRecoveryRole } = useGetMembersWithRecovery();
+  const shouldShowList = !isLoading && items && items.length;
+
+  const mappedItems = items.map((user) => {
+    const isChecked = checkedUsersList?.includes(user.walletAddress);
+
+    return {
+      user,
+      isChecked,
+    };
+  });
+
+  return (
+    <div>
+      <h3 className="text-1 mb-2">{title}</h3>
+      {isLoading && (
+        <div className="flex justify-center">
+          <SpinnerLoader appearance={{ size: 'medium' }} />
+        </div>
+      )}
+      {shouldShowList && (
+        <ul>
+          {mappedItems
+            .filter(notNull)
```

Comment:
There shouldn't be any filters added. The data that you will pass to this component should be prepared correctly outside this component.